src/lib.rs

```diff
@@ -122,10 +169,23 @@ pub fn validate() {
 /// Create an encryption key suitable for use with this project
 ///
 /// The encryption key will be written to the `keys.json` file at the root of your local secrets repository. You will need to commit this change yourself.
+#[no_mangle]
 pub fn generate_encryption_key() -> String {
     crate::encryption::generate_key()
 }
 
+/// Finds the `.configure` file in the current project and returns a string containing it.
+#[no_mangle]
+pub fn find_configuration_file() -> String {
+    match fs::find_configure_file() {
+        Ok(path) => match path.into_os_string().into_string() {
+            Ok(string) => string,
+            Err(_) => "".to_string(),
+        },
+        Err(_) => "".to_string(),
+    }
+}
```

Comment:
I haven't included it on the Ruby side (yet), but it was handy for debugging to know where the binary was looking for the `.configure` file. 

This could maybe become something like `bundle exec configure which` or similar? 🤔

Reply:
👍